cli/tests/functional/pom.xml

```diff
@@ -32,6 +32,18 @@
     <name>Helidon Command Line Interface Functional Tests</name>
 
     <dependencies>
+        <dependency>
```

Comment:
Can you add a comment to explain why this dependency is used ? (I.e. this is used to ensure that this module is always built after the `helidon-cli-maven-plugin`)